lib/r10k/module/svn.rb

```diff
@@ -74,18 +74,24 @@ def status
   end
 
   # @param [Hash] opts Deprecated
+  # @return [Boolean] true if the module was updated, false otherwise
   def sync(opts={})
+    updated = false
     if should_sync?
       case status
       when :absent
         install
+        updated = true
```

Comment:
IMO, this would be clearer if we did the `updated = true` as the first line after `if should_sync?` and then added an `else` clause where we set `updated = false`. It would also then match what we're doing in StatefulRepository.

Reply:
good call, I'll update that.

Reply:
updated, let me know what you think of what I ended up with.

Reply:
welp, looks like that doesn't work 😅 will fix

Reply:
not sure if I love the flipping back and forth that would happen if I set `updated = false` in an `else`, I might prefer what was there originally.. but I'll ruminate on that.